File: C/Sully.c
#include <stdio.h>
#include <string.h>
#include <stdlib.h>
int main(){
int i = 12;
if (i <= 0)
return(0);
char str[1000];
sprintf(str, "Sully_%d.c", i);
FILE *file = fopen(str, "w");
i--;
char *s="#include <stdio.h>%1$c#include <string.h>%1$c#include <stdlib.h>%1$cint main(){%1$cint i = %4$d;%1$cif (i <= 0)%1$creturn(0);%1$cchar str[5000];%1$csprintf(str, %2$cSully_%%d.c%2$c, i);%1$cFILE *file = fopen(str, %2$cw%2$c);%1$ci--;%1$cchar *s=%2$c%3$s%2$c;%1$cfprintf(file, s, 10, 34, s, i);%1$cfclose(file);%1$cchar command[5000];%1$csprintf(command, %2$cgcc -Wall -Wextra -Werror -o %%*s %%s%2$c, (int)strlen(str)-2, str, str);%1$csystem(command);%1$csprintf(command, %2$c./%%*s%2$c, (int)strlen(str)-2, str);%1$csystem(command);%1$c}%1$c";
fprintf(file, s, 10, 34, s, i);
fclose(file);
char command[1000];
sprintf(command, "gcc -Wall -Wextra -Werror -o %*s %s", (int)strlen(str)-2, str, str);
system(command);
sprintf(command, "./%*s", (int)strlen(str)-2, str);
system(command);
}

File: Python/Sully.py
import os

def main():
    i = 12
    if (i <= 0):
        return (0)
    str = ""
    str = "Sully_%d.py" % i
    f1 = open(str, "w")
    i -= 1
    s = r"print>>f1, 'import os\n\ndef main():\n\ti = 12\n\tif (i <= 0):\n\t\treturn (0)\n\tstr = \"\"\n\tstr = \"Sully_%d.c\" % i\n\tf1 = open(str, \"w\")\n\ti -= 1\n\ts = r\"' + s + '\"' + '\n\tf1.close()\n\tcommand = ""\n\tcommand = \"gcc -Wall -Wextra -Werror -o %*s %s\" % (str[:-2], str)\n\tos.system(command)\n\tcommand = \"./%s\" % \str[:-2]\n\texec(s)\nmain()'"
    f1.close()
    command = ""
    command = "gcc -Wall -Wextra -Werror -o %s %s" % (str[:-2], str)
    os.system(command)
    command = "./%s" % str[:-2]
    exec(s)
main()
